orchestra/utils/notifications.py

```diff
@@ -83,10 +84,32 @@ def notify_status_change(task, previous_status=None):
                                assignment.worker.user.email]
         }
 
+    if current_worker is None:
+        _notify_slack(task, current_worker)
+        _notify_email(task, message_info)
+
+    else:
```

Comment:
whooops!  just realized our model is a bit off and we have to rethink it a bit:
- we always notify a slack team/group about a task change, so slack isn't an option for workers to turn off
- so each communication preference type should have its list of applicable communicationtype choices, which are a subset of all of the choices